vpr/src/route/heap_type.h

```diff
@@ -191,6 +191,27 @@ class HeapInterface {
     // Note: Only invoke this method if all objects returned from this
     // HeapInterface instace have been free'd.
     virtual void free_all_memory() = 0;
+
+    // Set maximum number of elements that the heap should contain
+    // (the prune_limit).  If the prune limit is hit, then the heap should
+    // kick out duplicate index entries.
+    //
+    // For non-RCV routers this simply means taking the lowest cost entry for
+    // each index.  For RCV routers this is ???.
```

Comment:
For RCV routers we still want the lowest cost item in the heap (the router cost function is more complex, but we still want the lowest cost one).  So you can update or delete this comment (always want to keep the lowest cost items).

Reply:
I think it would be good to give some context of why this is necessary: basically what your wrote in the issue (can get many slightly better paths in some architectures, leading to the same node being on the heap many times -- if this happens a lot, memory usage can get out of control so we prune and save only the lowest cost entry on the heap for each node).

Reply:
Awesome, I've updated the comment.  Good to know that this will generalize to the RCV router too!